package.json

```diff
@@ -44,7 +44,7 @@
     "rxjs": "5.4.2",
     "style-loader": "0.18.2",
     "sw-precache-webpack-plugin": "0.11.4",
-    "transifex": "1.4.6",
+    "transifex": "git://github.com/mattkime/node-transifex.git#d109c7c",
```

Comment:
my changes are in node-transifex master, just bumped maintainer for npm release.